pylot/control/mpc/vehicle.py

```diff
@@ -0,0 +1,32 @@
+import numpy as np
+
+
+class Vehicle:
```

Comment:
Can we move this to the mpc/utils.py file and add comments?

Reply:
Done